bin/spring

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
Adding Spring (development preloader) could have been in a separate commit. I guess this is however fine.